supply-chain/src/components/graph_node.js

```diff
@@ -14,11 +14,22 @@ const GraphNode = (props) => {
     descriptions=null, images=null, highlighterFilter} = props;
   const meta = node in nodeToMeta ? nodeToMeta[node] : {};
 
+  const _isDesignProcessNode = () => {
+    if (meta.type === "process" && meta.stage_id === "S1") {
+      return true;
+    }
+    return false;
+  }
+
   const getHighlightClass = () => {
     if (node in highlights && highlights[node] !== 0) {
       return " highlighted " + getBackgroundGradient(highlights[node], highlights);
     } else if ((highlighterFilter === FILTER_CONCENTRATION) || (highlighterFilter === FILTER_COUNTRY)) {
       return " not-applicable";
+    } else if (highlighterFilter === FILTER_ORG && !_isDesignProcessNode()) {
```

Comment:
This is a bit fragile - we could add data for other nodes later (or remove it from these nodes). How much harder would it be to apply the styling based on whether data is present for the node or not?